app/models/dex_connector_ldap.rb

```diff
@@ -21,4 +19,5 @@ class DexConnectorLdap < ActiveRecord::Base
   validates :group_attr_user, presence: true
   validates :group_attr_group, presence: true
   validates :group_attr_name, presence: true
+  validates :certificate, presence: true
```

Comment:
Maybe not needed, as the validity is already checked in `javascript`, but this could also use the `x509_validator` (`x509_certificate: true`) used for the other certificates as a second check.

Reply:
As it is written currently it does not store the begin cert / end cert lines in the DB. I am not sure but I believe this will cause it not to pass x509 validity.

Agreed also that the check is done in JS, but I haven't checked all situations here. For example I believe if you enter valid values, hit the 'test connection' button, and then alter the values you can then hit save with bad values. That should be correct and is a bug I need to enter for the setup. This PR notably is to fix the fact that head is broken and unusable for external auth, not to correct all possible problems. That will be addressed in a subsequent change and PR.